### PR TITLE
feat(naming): make it easier to use custom names

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Would still require a lot of manual changes though as values are spread on many .yaml files, ie values.yaml product.yaml doc.yaml... A script would be the required to make all changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.

### CODE REVIEW

File: constants/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "constants"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+utils = {path = "../dependencies/control-plane/utils/utils-lib" }
+convert_case = "0.4.0"
```
Comment (sinhaashish) [inline constants/Cargo.toml:10]: a new line at end

**Review by sinhaashish — APPROVED** [review]
nit-pick , otherwise lgtm

File: constants/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+pub fn upgrade_job_img() -> String {
+    format!("{PRODUCT_NAME}-upgrade-job")
+}
+/// Upgrade job container image name.
+pub fn upgrade_job_container_name() -> String {
+    format!("{PRODUCT_NAME}-upgrade-job")
+}
+/// UPGRADE_EVENT_REASON is the reason field in upgrade job.
+pub fn upgrade_event_reason() -> String {
+    format!("{}Upgrade", product_pascal())
```
Comment (Abhinandan-Purkait) [inline constants/src/lib.rs:25]: Probably needs a space before Upgrade?

**Review by Abhinandan-Purkait — APPROVED** [review]

File: constants/src/lib.rs (hunk 2 above)
Comment (tiagolobocastro) [reply constants/src/lib.rs:25]: No, that's how it was "MayastorUpgrade"
> pub(crate) const UPGRADE_EVENT_REASON: &str = "MayastorUpgrade";

File: constants/Cargo.toml (hunk 1 above)
Comment (tiagolobocastro) [reply constants/Cargo.toml:10]: Other toml files here don't have a new line at the end, so I'd like to leave it the same for consistency

**Review by dsharma-dc — APPROVED** [review]

File: constants/src/lib.rs (hunk 2 above)
Comment (Abhinandan-Purkait) [reply constants/src/lib.rs:25]: Ahh okay.

Comment (Abhinandan-Purkait) [reply constants/src/lib.rs:25]: I believe the label is still `openebs.io/logging` for the loki scrape targets. We don't need to touch the scrape configs right?

Comment (tiagolobocastro) [reply constants/src/lib.rs:25]: That will have to be done manually, in fact in a bunch of places such as docs.yaml, chart,yaml, product.yaml and values.yaml in a few places.

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**